### PR TITLE
JIRAPLUGIN-163 - resolved issue design changes

### PR DESCRIPTION
**Pull request by daniel-toth-everit**

### CODE REVIEW

**Comment by balazs-zsoldos** [comment]
conflicts

File: src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java [hunk 1]
```diff
@@ -191,11 +197,12 @@ public EveritWorklog(final ResultSet rs)
     MutableIssue issueObject = issueManager.getIssueObject(issueId);
     issue = issueObject.getKey();
     issueSummary = issueObject.getSummary();
-
+    if (StatusCategory.COMPLETE
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java:200]: Use == for enumerators as it is faster and as Enumerator is instantiated only once in the JVM, it works.

File: src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java [hunk 2]
```diff
@@ -61,10 +62,14 @@
   private String exactRemaining;
 
   /**
+   * The issue closed.
+   */
+  private boolean isClosed = false;
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java:67]: isClosed is the name of the getter normally. The member variable is simply *closed"

File: src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java [hunk 3]
```diff
@@ -277,6 +284,10 @@ public String getExactRemaining() {
     return exactRemaining;
   }
 
+  public boolean getisClosed() {
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java]: would be isClosed() with the right variable name

File: src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java (hunk 1 above)
Comment (daniel-toth-everit) [inline src/main/java/org/everit/jira/timetracker/plugin/dto/EveritWorklog.java:200]: StatusCategory.COMPLETE is a String. And the check style don't like the == operator when comparing Strings.

**Pull request merged**